src/rand.c

```diff
@@ -171,23 +174,35 @@ int osrand_instantiate(void *vctx, unsigned int strength,
                        int prediction_resistance, const unsigned char *pstr,
                        size_t pstr_len, const OSSL_PARAM params[])
 {
+    OSRAND_RAND_CTX *ctx = (OSRAND_RAND_CTX *)vctx;
+    ctx->state = EVP_RAND_STATE_READY;
```

Comment:
It doesn't have to be true for lrng that might become blocking if there's not enough entropy but chance of that is pretty small so it's fine to just always return ready if device exist.